src/data/com_fri_1.json

```diff
@@ -43,7 +43,7 @@
     {
       "type": "paragraph",
       "data": {
-        "text": "K. Niech si\u0119 zmi\u0142uje nad nami B\u00f3g wszechmog\u0105cy *<br>i odpu\u015bciwszy nam grzechy, *<br>doprowadzi nas do \u017cycia wiecznego."
+        "text": "K. Niech si\u0119 zmi\u0142uje nad nami B\u00f3g wszechmog\u0105cy&nbsp;*<br>i odpu\u015bciwszy nam grzechy,&nbsp;*<br>doprowadzi nas do \u017cycia wiecznego."
```

Comment:
Since it will be used mostly on phones, I want to avoid this:
<img width="472" alt="Screen Shot 2021-08-13 at 21 38 06" src="https://user-images.githubusercontent.com/9083292/129410474-1388e2f9-9784-4e8c-b050-21831119a142.png">
and get this instead:
<img width="472" alt="Screen Shot 2021-08-13 at 21 38 16" src="https://user-images.githubusercontent.com/9083292/129410568-84c84de2-5fdb-4473-80d3-f926242bff04.png">